### DISCUSSION TITLE
Sorting albums by name

### DISCUSSION THREAD

**Discussion by rnbr04** [deckerst/aves#968] [Q&A] [answered]
Hi I know that there is an option to sort albums by name but the current sorting method begins sorting with the directory name

Is there any way to sort them with main folder name instead of beginning with directory? Or an option to hide the absolute directory naming of albums

**Comment by deckerst** [comment]
I don't understand. Could you provide an example with a few file paths, highlighting how it is sorted in Aves vs how you would like it to be?

**Comment by rnbr04** [comment]
![Screenshot_20240407-183748](https://github.com/deckerst/aves/assets/65329453/1a66d1cc-73bd-47cf-b3ed-2f4faedfc09b)

Like here for some albums the name begins with the root folder name (Backup), this disturbs the order I would want those albums to be in. For eg: "Backup/Download" should be displayed as just "Download" and placed after the albums beginning with letter B (Binsar... and Bluetooth)

**Comment by rnbr04** [comment]
I do have a folder named Download so maybe it is behaving this way in order to avoid duplicate album name, if my thought of process is correct then I don't have an issue with this behaviour since it can help me clear out the folders which contain similar kind of media

**Reply by deckerst** [reply, marked as answer]
Yes, as you guessed, the album names show the parent folder when the names would be otherwise ambiguous. It's dynamic, so if you rename one of these and it's no longer ambiguous, the names will no longer include the parent folder.

**Reply by rnbr04** [reply]
Thanks for the clarification and for this great app
